plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityEquipmentDropChance.java

```diff
@@ -0,0 +1,83 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.MapTag;
+import org.bukkit.entity.Mob;
+import org.bukkit.inventory.EntityEquipment;
+
+public class EntityEquipmentDropChance extends EntityProperty<MapTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name equipment_drop_chance
+    // @input MapTag
+    // @description
+    // Controls the chance of each piece of equipment dropping when the entity dies.
+    // Specifying a drop chance of 0 will prevent the item from dropping, a drop chance of 1 will always drop the item if killed by a player, and a drop chance of higher than 1 will always drop the item no matter what the entity was killed by.
+    //
+    // @Example
+    // # Use to prevent a zombie from dropping any of its equipped items, no matter what:
+    // - adjust <[zombie]> equipment_drop_chance:[helmet=0;chestplate=0;leggings=0;boots=0;main_hand=0;off_hand=0]

```

Comment:
`@mechanism-example` is a thing now, probably makes sense to use here seeing as this isn't relevant for the tag.
Or alternatively make it an example for both, I.e. setting it to `.drop_chance.with[leggings].as[0]` to make sure leggings don't drop while keeping other chances or something.